### PR TITLE
(SERVER-380) upgrade to jetty 9.2

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: test/unit/puppetlabs/services/config/puppet_server_config_service_test.clj [hunk 1]
```diff
@@ -26,6 +26,13 @@
 (def test-resources-dir
   "./dev-resources/puppetlabs/services/config/puppet_server_config_service_test")
 
+(def required-config
```
Comment (camlow325) [inline test/unit/puppetlabs/services/config/puppet_server_config_service_test.clj]: Should we also delete the other `required-config` from above?

**Comment by camlow325** [comment]
:+1: other than the removal of the extra `required-config` def.

**Comment by fpringvaldsen** [comment]
I'm also :+1: barring the removal of the extra `required-config`.

**Comment by cprice404** [comment]
Fixed

**Comment by nwolfe** [comment]
:+1:

**Pull request merged**